docs/send-data/opentelemetry-collector/use-case-prometheus.md

```diff
@@ -0,0 +1,162 @@
+---
+id: use-case-prometheus
+title: Monitoring Custom Metrics Using OpenTelemetry
+sidebar_label: Use case - Prometheus
+description: Learn how to import and monitor your existing Prometheus-formatted metrics into Sumo Logic.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+If you're using Prometheus, a popular tool for monitoring app performance, to collect metrics, it can be challenging to visualize and analyze that data in a single location. You can use our OpenTelemetry Collector to import your Prometheus-formatted metrics into Sumo Logic and create a custom dashboard for them.
```

Comment:
```suggestion
Prometheus is a popular tool for monitoring app performance. If you're using Prometheus to collect metrics, it can be challenging to visualize and analyze that data in a single location. You can use our OpenTelemetry Collector to import your Prometheus-formatted metrics into Sumo Logic and create a custom dashboard for them.
```

Reply:
I suggest you break up that first sentence.